### PR TITLE
Fix DetectionRandomAffine target-size to be in row-column format

### PR DESCRIPTION
**Pull request by cansik**
This fixed the `target-size` for DetectionRandomAffine as explained in #2011. It already takes into account if a specific `target-size` is set or the image shape is used.

### CODE REVIEW

File: src/super_gradients/training/transforms/transforms.py [hunk 1]
```diff
@@ -1510,7 +1510,7 @@ def random_affine(
 
     M = get_affine_matrix(img.shape[:2], target_size, degrees, translate, scales, shear)
 
-    img = cv2.warpAffine(img, M, dsize=target_size, borderValue=(border_value, border_value, border_value))
+    img = cv2.warpAffine(img, M, dsize=target_size[::-1], borderValue=(border_value, border_value, border_value))
```
Comment (BloodAxe) [inline src/super_gradients/training/transforms/transforms.py:1513]: I suggest to add an explicit message to a method's docstring to indicate whether `target_size` represents size as (rows, cols) or (cols, rows).
And unpack these to add more clarity to the code. Eg:
```
rows, cols = target_size[:2]
img = cv2.warpAffine(img, M, dsize=(cols, rows), ...)

```

Comment (cansik) [reply src/super_gradients/training/transforms/transforms.py]: Good point, I've adjusted it.

**Review by BloodAxe — APPROVED** [review]
LGTM

**Comment by BloodAxe** [comment]
Thank you for your contribution! Merged to master 🎉

**Pull request merged**